e2e/cypress/integration/account_settings/display/account_settings_positiion_spec.js

```diff
@@ -0,0 +1,99 @@
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @account_setting
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+describe('Account Settings > General > Position', () => {
+    let testTeam;
+    let testUser;
+
+    beforeEach(() => {
+        cy.apiAdminLogin();
+        cy.apiInitSetup().then(({team, user}) => {
+            testTeam = team;
+            testUser = user;
+        });
+    });
+
+    it('MM-T2063 Position / Can enter position', () => {
+        cy.apiLogin(testUser);
+
+        // # Visit town square
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Open the hamburger menu
+        cy.findByLabelText('main menu').should('be.visible').click();
+
+        // # Click on Account settings menu item
+        cy.findByText('Account Settings').should('be.visible').click();
+
+        // # Fill-in the position field
+        cy.findByText('Position').should('be.visible').click();
+        cy.get('#position').type('Master hacker');
+        cy.get('#saveSetting').click();
+
+        // # Exit the modal
+        cy.get('body').type('{esc}', {force: true});
+
+        // # Post message in the main channel
+        cy.postMessage('hello from master hacker');
+
+        // # Click on the profile image
+        cy.get('.profile-icon > img').as('profileIconForPopover').click();
+
+        // # Verify that the popover is visible
+        cy.get('#user-profile-popover').should('be.visible').within(() => {
+            // * and contains the entered position
+            cy.findByText('Master hacker').should('be.visible');
+        });
+    });
+
+    it('MM-T2064 Position / 128 characters', () => {
+        const position = 'masterhackermasterhackermasterhackermasterhackermasterhackermasterhackermasterhackermasterhackermasterhackermasterhackermasterha';
+
+        cy.apiLogin(testUser);
+
+        // # Visit town square
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Open the hamburger menu
+        cy.findByLabelText('main menu').should('be.visible').click();
+
+        // # Click on Account settings menu item
+        cy.findByText('Account Settings').should('be.visible').click();
+
+        // # Fill-in the position field with a value of 128 characters
+        cy.findByText('Position').should('be.visible').click();
+        cy.get('#position').type(position);
+        cy.get('#saveSetting').click();
+
+        cy.get('#position').invoke('val').then((val) => {
+            // # Verify that the input value is 128 characters
+            expect(val.length).to.equal(128);
+        });
```

Comment:
I'm getting error here. Should this move before saving?
<img width="758" alt="Screen Shot 2020-11-24 at 7 22 40 PM" src="https://user-images.githubusercontent.com/5334504/100089247-5a498980-2e8c-11eb-86c6-5242f1d7f354.png">

Reply:
Still failed on me. Could you please check if it's passing on you? Thanks
![Screen Shot 2020-12-07 at 7 09 09 PM](https://user-images.githubusercontent.com/5334504/101343963-b914f700-38bf-11eb-95bd-10b0d5579d80.png)